coordinator/points_writer.go

```diff
@@ -153,10 +152,10 @@ func (w *PointsWriter) Close() error {
 	return nil
 }
 
-// SetLogOutput sets the writer to which all logs are written. It must not be
-// called after Open is called.
-func (w *PointsWriter) SetLogOutput(lw io.Writer) {
-	w.Logger = log.New(lw, "[write] ", log.LstdFlags)
+// WithLogger sets the logger to augment for log messages. It must not be
+// called after the Open method has been called.
```

Comment:
nit: again, not your change, but if we really shouldn't call this after we open the pointswriter, then this call should be defensive and simply not allow it, panic, or something.  The way it is currently written the comment is really useless as no one will see that if they use the method.